tests/templates/kuttl/ldap/05-create-authentication-classes.yaml.j2

```diff
@@ -1,69 +1,77 @@
-{% if test_scenario['values']['ldap-authentication'] == 'no-tls' %}
 ---
-apiVersion: authentication.stackable.tech/v1alpha1
-kind: AuthenticationClass
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
 metadata:
-  name: airflow-with-ldap-no-tls-ldap
-spec:
-  provider:
-    ldap:
-      hostname: openldap.default.svc.cluster.local
-      port: 1389
-      searchBase: ou=users,dc=example,dc=org
-      bindCredentials:
-        secretClass: airflow-with-ldap-bind
+  name: openldap
+commands:
+  - script: |
+      kubectl apply -n $NAMESPACE -f - <<EOF
+      ---
+      apiVersion: secrets.stackable.tech/v1alpha1
+      kind: SecretClass
+      metadata:
+        name: airflow-with-ldap-bind
+      spec:
+        backend:
+          k8sSearch:
+            searchNamespace:
+              pod: {}
+{% if test_scenario['values']['ldap-authentication'] == 'no-tls' %}
+      ---
+      apiVersion: authentication.stackable.tech/v1alpha1
+      kind: AuthenticationClass
+      metadata:
+        name: airflow-with-ldap-no-tls-ldap-$NAMESPACE
```

Comment:
This name will surely not get too long? :thinking:

Reply:
Yes, that was in the back of my mind too! I'll shorten them

Reply:
Have changed this and run the ldap tests on Openshift again:

```
--- PASS: kuttl (640.94s)
    --- PASS: kuttl/harness (0.00s)
        --- PASS: kuttl/harness/ldap_airflow-latest-2.4.1-stackable23.4.0-rc3_ldap-authentication-insecure-tls_openshift-true (197.82s)
        --- PASS: kuttl/harness/ldap_airflow-latest-2.4.1-stackable23.4.0-rc3_ldap-authentication-server-verification-tls_openshift-true (226.46s)
        --- PASS: kuttl/harness/ldap_airflow-latest-2.4.1-stackable23.4.0-rc3_ldap-authentication-no-tls_openshift-true (206.32s)

```